SparkPost/support-docs
# EA-9272 - Add pages under analyst SFMC section (and fix other articles)

### What Changed

- Added pages for SFMC section of `/analyst` pages.
- Updated existing pages to fix links, add missing pages, and remove broken things

### How To Test or Verify

- Check new pages under the `/analyst/sfmc` section exist and function.
- Check existing pages for broken links or non-existent videos/pictures.
- No references to intercom or support.emailanalyst.com

### PR Checklist

#### All PRs Checklist


#### Content Changes Checklist
